packages/mattermost-redux/src/selectors/entities/posts.ts

```diff
@@ -484,7 +495,7 @@ export const getLastPostPerChannel: (state: GlobalState) => RelationOneToOne<Cha
             }
 
             const postId = recentBlock.order[0];
-            if (allPosts.hasOwnProperty(postId)) {
+            if (Object.prototype.hasOwnProperty.call(allPosts, postId)) {
```

Comment:
Not something we need to change here, but I think we might want to disable this ESLint warning. It's only really useful if you're worried about people overwriting `hasOwnProperty` or messing up the prototype of an object, and I think we can probably agree that this syntax is uglier than the old one.

Reply:
Reverted ↩️